cmd/queue/main.go

```diff
@@ -328,6 +362,16 @@ func main() {
 		logger.Fatal(err.Error())
 	}
 
+	// If this is set, we run as a standalone binary to probe the queue-proxy.
```

Comment:
What is this change for? What's the `standalone binary` here?

Reply:
the binary doesnt have anything to do with this pr. I just moved the bit for logging above it to have access to the logger. but git is confused and marks other bits as changed code.